### PR TITLE
Update "v5.4-performance-benchmarking-with-tpch.md“

### PR DESCRIPTION
**Pull request by en-jin19**
### What is changed, added or deleted? (Required)
Fix some parts’ format in "v5.4-performance-benchmarking-with-tpch.md".

- Since most Chinese and English docs in "benchmark" folder has no "summary" parts, this PR has removed the part as well. If needed, I will add the parts to all docs in the folder later.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v6.0 (TiDB 6.0 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/8601
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by en-jin19** [comment]
Hi @heipark, could you please tech review this PR? thanks!

**Review by heipark — APPROVED** [review]

File: benchmark/v5.4-performance-benchmarking-with-tpch.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
 ---
-title: TiDB TPC-H 100GB Performance Test Report -- v5.4 MPP mode vs. Greenplum 6.15.0 and Apache Spark 3.1.1
-summary: Compare the TPC-H 100GB performance of TiDB 5.4 MPP mode, Greenplum 6.15.0, and Apache Spark 3.1.1.
+title: TiDB TPC-H Performance Test Report -- v5.4 MPP mode vs. Greenplum 6.15.0 and Apache Spark 3.1.1
+summary: Compare the TPC-H 100 GB performance of TiDB 5.4 MPP mode, Greenplum 6.15.0, and Apache Spark 3.1.1.
```
Comment (en-jin19) [inline benchmark/v5.4-performance-benchmarking-with-tpch.md:3]:
```suggestion
title: TiDB TPC-H Performance Test Report -- v5.4 MPP mode vs. Greenplum 6.15.0 and Apache Spark 3.1.1
```

**Review by shichun-0415 — APPROVED** [review]
LGTM

**Pull request merged**